### DISCUSSION TITLE
Blackhole won't turn off

### DISCUSSION THREAD

**Discussion by drstrange146** [ExistentialAudio/BlackHole#609] [Q&A]
Love the program, everything I needed it for works great. However, I can't seem to _not_ use it, and it's interfering with chat apps I use when I'm not trying to share desktop audio along with my voice.

I have my audio midi setup switched back to built-in mic/output, and muted blackhole 2ch in its own source and in the multi-output, just in case. Still, when I use apps like discord or google hangouts it sends feedback to the other people I'm talking to of all other voices/bot voices--hence assuming it's my desktop audio that they're getting.

I also made sure that those apps were looking at the built-in sources or the default sources for their ins and outs, but they're still picking up computer audio. It happens when I'm wearing headphones too, so it's not a feedback issue from roomtone. It also doesn't eventually turn into a high pitched feedback tone. Running Big Sur. No other apps open at the time. Tried resetting audio driver cache in terminal and doing a hard restart.

Any advice?

**Comment by devinroth** [comment]
This is definitely not an issue with BlackHole. Those apps are clearly screwing something up. Please contact them.
